pandunia-lekse.csv

```diff
@@ -1254,6 +1267,8 @@ komuniste||||communist|||||||||||||||komunisto|kommunisti|komunista
 komunistia||||communism|||||||||||||||komunismo|kommunismi|komunizm
 konakri ginia|dex|GN||Guinea||||||||||||||||Guinea|Gwinea
 koncung||||insect|insecte|insecto|inseto|насекомое||昆虫|昆虫|곤충|sâu bọ (côn trùng)|कीड़ा ||serangga|mdudu|böcek||hyönteinen (ötökkä)|owad
+konfusi|||eng:confused, spa:confundido, por:confuso, zho:困惑 (kùnhuò)|confused|perplexe (désorienté)|confundido|perplexo (confuso)|||困惑 (混乱)|まごまご||||||||||
+konfusu||||confuse (perplex)||confundir|||||||||||||||
```

Comment:
harfe -s- sa mede da vokal laya -z- sa panduni. yi lekse xudu esa "konfuzi" e "konfuzu".

Reply:
si, a "konfuze" lew mena "Confucius". we okey?